$FreeBSD$

Unfortunately we can't do the usual patch configure.ac and run autoconf -- the
tarball doesn't contain the autotools sources ><

--- configure.orig	2011-08-23 18:18:22.000000000 +0400
+++ configure	2011-08-30 22:27:42.000000000 +0400
@@ -5104,105 +5104,6 @@
 
 fi
 
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for cvSobel in -lcv" >&5
-$as_echo_n "checking for cvSobel in -lcv... " >&6; }
-if test "${ac_cv_lib_cv_cvSobel+set}" = set; then :
-  $as_echo_n "(cached) " >&6
-else
-  ac_check_lib_save_LIBS=$LIBS
-LIBS="-lcv  $LIBS"
-cat confdefs.h - <<_ACEOF >conftest.$ac_ext
-/* end confdefs.h.  */
-
-/* Override any GCC internal prototype to avoid an error.
-   Use char because int might match the return type of a GCC
-   builtin and then its argument prototype would still apply.  */
-#ifdef __cplusplus
-extern "C"
-#endif
-char cvSobel ();
-int
-main ()
-{
-return cvSobel ();
-  ;
-  return 0;
-}
-_ACEOF
-if ac_fn_c_try_link "$LINENO"; then :
-  ac_cv_lib_cv_cvSobel=yes
-else
-  ac_cv_lib_cv_cvSobel=no
-fi
-rm -f core conftest.err conftest.$ac_objext \
-    conftest$ac_exeext conftest.$ac_ext
-LIBS=$ac_check_lib_save_LIBS
-fi
-{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_cv_lib_cv_cvSobel" >&5
-$as_echo "$ac_cv_lib_cv_cvSobel" >&6; }
-if test "x$ac_cv_lib_cv_cvSobel" = x""yes; then :
-  cat >>confdefs.h <<_ACEOF
-#define HAVE_LIBCV 1
-_ACEOF
-
-  LIBS="-lcv $LIBS"
-
-else
-
-    echo "***"
-    echo "** Fatal Error : Header or library files for 'OpenCV (core)' are missing but are mandatory to compile G'MIC. **"
-    echo "***"
-    exit -1
-
-fi
-
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for main in -lhighgui" >&5
-$as_echo_n "checking for main in -lhighgui... " >&6; }
-if test "${ac_cv_lib_highgui_main+set}" = set; then :
-  $as_echo_n "(cached) " >&6
-else
-  ac_check_lib_save_LIBS=$LIBS
-LIBS="-lhighgui  $LIBS"
-cat confdefs.h - <<_ACEOF >conftest.$ac_ext
-/* end confdefs.h.  */
-
-
-int
-main ()
-{
-return main ();
-  ;
-  return 0;
-}
-_ACEOF
-if ac_fn_c_try_link "$LINENO"; then :
-  ac_cv_lib_highgui_main=yes
-else
-  ac_cv_lib_highgui_main=no
-fi
-rm -f core conftest.err conftest.$ac_objext \
-    conftest$ac_exeext conftest.$ac_ext
-LIBS=$ac_check_lib_save_LIBS
-fi
-{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_cv_lib_highgui_main" >&5
-$as_echo "$ac_cv_lib_highgui_main" >&6; }
-if test "x$ac_cv_lib_highgui_main" = x""yes; then :
-  cat >>confdefs.h <<_ACEOF
-#define HAVE_LIBHIGHGUI 1
-_ACEOF
-
-  LIBS="-lhighgui $LIBS"
-
-else
-
-    echo "***"
-    echo "** Fatal Error : Header or library files for 'OpenCV (highgui)' are missing but are mandatory to compile G'MIC. **"
-    echo "***"
-    exit -1
-
-fi
-
-
 # Checks for header files.
 ac_ext=c
 ac_cpp='$CPP $CPPFLAGS'
